database/mysql_seeder.sql

```diff
@@ -0,0 +1,25 @@
+DROP DATABASE IF EXISTS images_service;
+CREATE DATABASE IF NOT EXISTS images_service;
+
+USE images_service;
+
+
+
+CREATE TABLE IF NOT EXISTS product_images (
+  id int NOT NULL AUTO_INCREMENT,
+  product_id int NOT NULL,
+  s3_url VARCHAR (255) NOT NULL,
+  PRIMARY KEY (id)
+);
+
+LOAD DATA LOCAL INFILE '/Users/sinamb/Documents/Programs/images-service/data.csv'
```

Comment:
I would recommend using path here to determine the path to the file as it will be different once you deploy.